plugin/storage/badger/README.md

```diff
@@ -38,4 +38,8 @@ Exception to the above is the duration index, since there are no exact duration
 
 Because each TraceID is stored as spans, the same TraceID can appear multiple times from a index query. Other than duration query, this means they are coming in order so each of them is discarded by easily checking if the previous one is equal to current one, but with the duration index the spans can come in random order and thus hash-join is used to filter the duplicates.
 
-After all the index keys have been scanned, the process is then sent to the merge-join where two index queries are compared and only matching IDs are taken. After that, the next one is compared to the result of the previous and so forth until all the index fetches have been processed. The resulting query set is the list of TraceIDs that matched all the requirements. 
\ No newline at end of file
+After all the index keys have been scanned, the process is then sent to the merge-join where two index queries are compared and only matching IDs are taken. After that, the next one is compared to the result of the previous and so forth until all the index fetches have been processed. The resulting query set is the list of TraceIDs that matched all the requirements.
+
+## Upgrade Badger v1 to v3
```

Comment:
This does not need to be in the README, it is not the information everyone needs. Let's move it to a separate .md file